src/types/lazy.go

```diff
@@ -121,8 +121,12 @@ func wrap(typ byte, byteFields []uint8, args ...string) string {
 	return string(buf)
 }
 
-func unwrap1(s string) (one string) {
-	return s[stringLenBytes+1:] // do not care about length, there is only 1 param
+func unwrap1(s string) string {
```

Comment:
tell me what problem you're solving.

Reply:
Check 'IsBoolean' in `map.go`. I wanna check can we unwap bool element.